tests/integration/spec.bats

```diff
@@ -79,15 +79,15 @@ function teardown() {
 
   SPEC_REF=$([[ -z "$SPEC_COMMIT" ]] && echo $SPEC_VERSION || echo $SPEC_COMMIT)
 
-  run git -C src/runtime-spec reset --hard "${SPEC_REF}"
+  run bash -c "cd src/runtime-spec && git reset --hard ${SPEC_REF}"
```

Comment:
What is the reason for this change? I guess old(er) git does not support `-C`?

Reply:
Yes, CentOS 7 is shipped with an ancient version of git (`git version 1.8.3.1`).